app/models/naming.rb

```diff
@@ -478,8 +478,8 @@ def self.all_reasons
     end
 
     # Initialize Reason.
-    def initialize(reasons, num)
-      @reasons = reasons
+    def initialize(reason, num)
```

Comment:
I was looking for a better name, thanks. It was super confusing to figure out what this even was, because there was another ivar in the view, iirc, with a similar and similarly misleading name, and when i finally figured that out I was on a rampage to flip the names and restore order in the universe.

Reply:
Will do this later when i can concentrate!

Reply:
Done